test/e2e_test.go

```diff
@@ -156,6 +154,7 @@ func (e *e2e) TestSecurityProfilesOperator() {
 			tc.fn(nodes)
 		})
 	}
+	e.cleanupOperator(manifest)
```

Comment:
I think it should be `namespaceManifest` instead of `manifest` here.

Reply:
Yep, fixed :+1: